lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -48,8 +54,45 @@ table.ccx-schedule {
 
 form.ccx-form {
     line-height: 1.5;
+    // inspiration was taken from https://github.com/edx/ux-pattern-library
     select {
         width: 100%;
+        height: ($baseline*1.9);
```

Comment:
This seems like a Magic Number (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#imperfect-and-magic-numbers). This should be avoided in general and with this particular element, a select field, I'd prefer no height be set (know that the UXPL currently does this).